lib/internal/child_process.js

```diff
@@ -545,7 +551,8 @@ function setupChannel(target, channel) {
       options = undefined;
     } else if (options !== undefined &&
                (options === null || typeof options !== 'object')) {
-      throw new errors.TypeError('ERR_INVALID_ARG_TYPE', 'options', 'Object');
+      throw new errors.TypeError('ERR_INVALID_ARG_TYPE', 'options', 'Object',
```

Comment:
Yeah I noticed but few lines above (260) it is in lowercase. I'd make them consistent.